managed_vms/jetty9-gcloud-sessions/src/main/jetty-base/etc/jetty-gcloud-sessions.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <!-- ============================================================================================== -->
+  <!-- GCloud configuration.                                                                          -->
+  <!-- This replaces the file in jetty releases < 9.3.8 with a fix to extract the worker name from    -->
+  <!-- the GAE_MODULE_INSTANCE env variable.  This can be removed from jetty-9.3.8 onwards.           -->
+  <!-- ============================================================================================== -->
+  <New id="gconf" class="org.eclipse.jetty.gcloud.session.GCloudConfiguration">
+    <!-- To contact remote gclouddatastore set the following properties in start.ini -->
+    <!-- Either set jetty.gcloudSession.projectId or use system property/env var DATASTORE_DATASET-->
+    <Set name="projectId"><Property name="jetty.gcloudSession.projectId"/></Set>
+    <Set name="p12File"><Property name="jetty.gcloudSession.p12File"/></Set>
```

Comment:
All Google services are moving to [Usable Auth](https://github.com/GoogleCloudPlatform/gcloud-java#authentication) for 2LO (computer<==>computer).  (It automatically does auth in AppEngine, GCP (MVMs, GCE, GKE, inside or outside of Docker containers, etc), and locally.)

Datastore currently uses this, and the way we'll be doing SQL will also require something like this. (as well as MySQL username/pw).

Understandable that you don't support it yet, but it's the direction we are going and we are trying to eliminate the need for files, json or p12 except when you are running outside the environment in which case you can set an environment variable to point to your json credentials.

For the main thrust of your question -- I support going forward with what you have, but we do need to get it to move towards the new APIs / auth. ( @ludo call)